<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2579</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Monitor &amp; Recorder</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_camera_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>642</width>
      <height>362</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_1">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera #1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_camera_2">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>0</y>
      <width>642</width>
      <height>362</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_2">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>1</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera #2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_camera_3">
    <property name="geometry">
     <rect>
      <x>1285</x>
      <y>0</y>
      <width>642</width>
      <height>362</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_3">
    <property name="geometry">
     <rect>
      <x>1286</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera #3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_camera_4">
    <property name="geometry">
     <rect>
      <x>1927</x>
      <y>0</y>
      <width>642</width>
      <height>362</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_4">
    <property name="geometry">
     <rect>
      <x>1927</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera #4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_camera_open_1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_camera_open_2">
    <property name="geometry">
     <rect>
      <x>1184</x>
      <y>370</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_camera_open_3">
    <property name="geometry">
     <rect>
      <x>1826</x>
      <y>370</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_camera_open_4">
    <property name="geometry">
     <rect>
      <x>2468</x>
      <y>370</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2579</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
